pkg/services/sqlstore/datasource_test.go

```diff
@@ -3,173 +3,212 @@ package sqlstore
 import (
 	"testing"
 
-	. "github.com/smartystreets/goconvey/convey"
-
 	"github.com/grafana/grafana/pkg/models"
+	"github.com/stretchr/testify/assert"
 )
 
-type Test struct {
-	Id   int64
-	Name string
-}
-
 func TestDataAccess(t *testing.T) {
-	Convey("Testing DB", t, func() {
-		InitTestDB(t)
-		Convey("Can add datasource", func() {
+	defaultAddDatasourceCommad := models.AddDataSourceCommand{
```

Comment:
Wanted to add tests but also decided to rewrite this to go testing instead of convey

Reply:
Great! :)